src/pages/ResetPassword/2-CodeRequest/index.tsx

```diff
@@ -25,6 +26,11 @@ const CodeRequest: React.FC = () => {
     const resetBody = { email };
     try {
       await authApi.requestPasswordResetCode(resetBody);
+      setInfo({
+        ...informationAboutCodeValidation,
+        message: "Código reenviado. Verifique a sua caixa de entrada.",
+        type: "successful"
```

Comment:
Talvez ao invés de `type`, `status` faça mais sentido.  Só colocando aqui já que já to fazendo a review do resto mesmo, mas type ta bom também